asm/asmRISCV/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>asmRSICV</artifactId>
+	<packaging>jar</packaging>
+	<name>ASM RSICV grammar</name>
+	<parent>
+		<groupId>org.antlr.grammars</groupId>
+		<artifactId>asmparent</artifactId>
```

Comment:
This looks ok, but the *parent* pom.xml, the one at .../asm/pom.xml, does not contain a link to this pom.xml, .../asm/asmriscv/pom.xml. It needs to be added:
```
01/23-07:07:52 ~/test/grammars-v4/asm
$ mvn clean test
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] Assembler Grammars                                                 [pom]
[INFO] ASM 6502 grammar                                                   [jar]
[INFO] ASM 8080 grammar                                                   [jar]
[INFO] ASM 8086 grammar                                                   [jar]
[INFO] MASM grammar                                                       [jar]
[INFO] ASM Z80 grammar                                                    [jar]
[INFO] MASM grammar                                                       [jar]
[INFO] NASM grammar                                                       [jar]
[INFO] ASM pdp7 grammar                                                   [jar]
[INFO] CUDA PTX ISA Grammars                                              [pom]
[INFO] CUDA PTX ISA 1.0 grammar                                           [jar]
[INFO] CUDA PTX ISA 2.1 grammar                                           [jar]
[INFO]
[INFO] --------------------< org.antlr.grammars:asmparent >--------------

...

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Assembler Grammars 1.0-SNAPSHOT:
[INFO]
[INFO] Assembler Grammars ................................. SUCCESS [  0.341 s]
[INFO] ASM 6502 grammar ................................... SUCCESS [  6.616 s]
[INFO] ASM 8080 grammar ................................... SUCCESS [  0.878 s]
[INFO] ASM 8086 grammar ................................... SUCCESS [  1.592 s]
[INFO] MASM grammar ....................................... SUCCESS [  1.135 s]
[INFO] ASM Z80 grammar .................................... SUCCESS [  0.601 s]
[INFO] MASM grammar ....................................... SUCCESS [  0.730 s]
[INFO] NASM grammar ....................................... SUCCESS [  4.373 s]
[INFO] ASM pdp7 grammar ................................... SUCCESS [  3.268 s]
[INFO] CUDA PTX ISA Grammars .............................. SUCCESS [  0.002 s]
[INFO] CUDA PTX ISA 1.0 grammar ........................... SUCCESS [  1.161 s]
[INFO] CUDA PTX ISA 2.1 grammar ........................... SUCCESS [ 20.722 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  41.661 s
[INFO] Finished at: 2024-01-23T07:08:42-05:00
[INFO] ------------------------------------------------------------------------
01/23-07:08:42 ~/test/grammars-v4/asm
$ ls
asm6502/  asm8080/  asm8086/  asmMASM/  asmriscv/  asmZ80/  masm/  nasm/  pdp7/  pom.xml  ptx/
01/23-07:09:06 ~/test/grammars-v4/asm
$ grep asmriscv pom.xml
01/23-07:09:15 ~/test/grammars-v4/asm
$
```